init.zsh

```diff
@@ -86,6 +86,9 @@ function pmodload {
 # Prezto Initialization
 #
 
+# Find the dir Prezto is installed to
+ZPREZTODIR=${ZPREZTODIR:-${ZDOTDIR:-$HOME}/.zprezto}
```

Comment:
Your explanation in the PR makes sense, but also belongs here in a comment. [A good comment tells you _why_ something is done.](https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/)

Would it be better to use `${0:h}`? Introspected, rather than injected. That way, if Prezto is in an unusual place, no configuration is needed.